app/controllers/api/v1/targets_controller.rb

```diff
@@ -1,6 +1,10 @@
 module Api
   module V1
     class TargetsController < Api::V1::ApiController
+      def index
+        @targets = policy_scope(current_user.targets)
```

Comment:
model User doesn't seems to have a `targets` association, are you missing something?

Reply:
Idk what do you mean. Maybe you are referring to use `has_many :targets, dependent: :destroy` in the `User` model to 
establish an association? If that is the case, I am currently using it.